core/factory.py

```diff
@@ -133,7 +141,18 @@ def notify_about_membership(self, message) -> None:
         finally:
             # if user is not a member, using the bot is considered violation
             self.set_context(message, 'violation', True)
-
+    
+    def notify_about_violation(self, message) -> None:
+        try:
+            # delete sent message as a channel
+            self.delete_message(message.chat.id, message.message_id)
+            # notify user about this
+            self.send_message(message.from_user.id, self.strings.you_are_blogger)
```

Comment:
It seems `from_user.id` points to a bot called Channel, we can't send message to it. I don't think we can send message at all in this case. Let's remove that part only delete the message for now.